presto-orc/src/main/java/com/facebook/presto/orc/Filters.java

```diff
@@ -34,7 +35,7 @@
 
     private Filters() {}
 
-    private static class AlwaysFalse
+    public static class AlwaysFalse
```

Comment:
@oerling AlwaysFalse, IsNull and IsNotNull classes should stay private. Use `notNull().equals(filter)` instead of `filter instanceof IsNotNull`.